### ISSUE TITLE
document that "In Private" mode of your browser mode should be used if you'll be switching between HoloLens 1st Gen and HoloLens 2 devices

### ISSUE THREAD

**Issue by ForrestTrepte** [MicrosoftDocs/mixed-reality#1655]
I recommend adding the following info to the "Connecting over USB" section:

If you access the device portal with a HoloLens 1st gen device and then access device portal with a HoloLens 2 via http://127.0.0.1:10080, the controls will disappear.

In order to work around this, you can use the hamburger menu's Reset Layout command to get the controls back or you can use the "In Private" mode of your browser.

This issue occurs because the browser will cache the layout of device portal and there are differences between 1st gen and V2 HoloLens device portal layouts.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37e0e5f1-b23b-4909-c1ba-6dba39174d30
* Version Independent ID: b0a8777f-e04e-b645-d983-0acb94268f4c
* Content: [Using the Windows Device Portal - Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-windows-device-portal)
* Content Source: [mixed-reality-docs/using-the-windows-device-portal.md](https://github.com/MicrosoftDocs/mixed-reality/blob/master/mixed-reality-docs/using-the-windows-device-portal.md)
* Product: **mixed-reality**
* Technology: **mr-dev**
* GitHub Login: @JonMLyons
* Microsoft Alias: **jlyons**

**Comment by jessemcculloch** [comment]
@ForrestTrepte - Has this been reported as a bug to the engineering teams?

**Comment by ForrestTrepte** [comment]
Yes, I entered it in feedback hub at https://aka.ms/AA66c4l. The resulting bug in the OS database was resolved By Design. "This is actually known. The browser will cache the layout of device portal and there are differences between HLv1 and HLv2 device portal layouts, hence the black screen issue.
Workaround is to use "In Private" mode on the browser if you need to switch between devices."

This doesn't seem like a good design for user experience, but perhaps there is an architectural reason that this is difficult to fix and folks will eventually be transitioning to HoloLens 2 instead of switching back and forth? If this can be fixed it would be greatly preferable, but if not then I recommend documenting this workaround.
